bazel/google_cloud_cpp_spanner_deps.bzl

```diff
@@ -50,28 +50,28 @@ def google_cloud_cpp_spanner_deps():
         )
 
     # Load the googleapis dependency.
-    if "com_github_googleapis_googleapis" not in native.existing_rules():
+    if "com_google_googleapis" not in native.existing_rules():
         http_archive(
-            name = "com_github_googleapis_googleapis",
+            name = "com_google_googleapis",
             urls = [
-                "https://github.com/google/googleapis/archive/6a3277c0656219174ff7c345f31fb20a90b30b97.zip",
+                "https://github.com/google/googleapis/archive/32a10f69e2c9ce15bba13ab1ff928bacebb25160.zip",
             ],
-            strip_prefix = "googleapis-6a3277c0656219174ff7c345f31fb20a90b30b97",
-            sha256 = "82ba91a41fb01305de4e8805c0a9270ed2035007161aa5a4ec60f887a499f5e9",
-            build_file = "@com_github_googleapis_google_cloud_cpp//bazel:googleapis.BUILD",
+            strip_prefix = "googleapis-32a10f69e2c9ce15bba13ab1ff928bacebb25160",
+            sha256 = "2c5dc6d4575e0804ccf3ffb22ca36d68621013c0b2980042d6fb4a04a4447b17",
+            build_file = "//bazel:googleapis.BUILD",
         )
 
     # Load gRPC and its dependencies, using a similar pattern to this function.
     # This implictly loads "com_google_protobuf", which we use.
     if "com_github_grpc_grpc" not in native.existing_rules():
         http_archive(
             name = "com_github_grpc_grpc",
-            strip_prefix = "grpc-1.19.1",
+            strip_prefix = "grpc-1.21.0-pre1",
```

Comment:
Do you really want to depend on a pre-release?  Should we wait a couple of days and depend on the actual release?

Reply:
updated to grpc-1.21.0